modules/end-user-guide/partials/ref_parameters-for-che-editor-yaml.adoc

```diff
@@ -7,7 +7,7 @@ The simplest way to select an IDE in the `che-editor.yaml` is to specify the `id
 ====
 [source,yaml]
 ----
-id: che-incubator/che-code/insiders
+id: che-incubator/che-idea/latest
 ----
 ====
```

Comment:
Example block, then code block

```suggestion
====
----
```

Reply:
:) Thanks, but that's  the closing syntax.